### PR TITLE
Troubleshooting pages

### PR DESCRIPTION
**Pull request by marianhlavac**
This is a proposal for troubleshooting pages.

🤝 This pull request also stands for proposal to create troubleshooting topics for all tutorial pages. Each tutorial should have troubleshooting page containing frequently asked questions or problems (at least some of them), and solutions to them.

### CODE REVIEW

File: troubleshooting/index.adoc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+= Troubleshooting
+Marián Hlaváč <marian.hlavac@fit.cvut.cz>
```
Comment (hroncok) [inline troubleshooting/index.adoc]: do we generally attribute individual authors?

File: troubleshooting/index.adoc [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+Pokud nenosíte v peněžence čtyřlístek, nebo si na cvičení nenosíte talisman pro štěstí, pravděpodobně se stane, že se něco nepovede nebo se na něčem zaseknete.
+
+V této sekci naleznete přehledně nejčastější problémy a jejich řešení, formou Q&A.
+Problémy jsou rozděleny do tématických celků.
```
Comment (hroncok) [inline troubleshooting/index.adoc:6]: dal bych sem `:toc:`

File: troubleshooting/index.adoc [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+== Oblasti problémů
+
+TODO: Zde budou odkazy na jednotlivé kategorie problémů a jejich stručný popis
```
Comment (hroncok) [inline troubleshooting/index.adoc]: WARNING:

File: troubleshooting/index.adoc (hunk 1 above)
Comment (marianhlavac) [reply troubleshooting/index.adoc]: On this page it's mistake, but otherwise - why not?

File: troubleshooting/index.adoc (hunk 2 above)
Comment (marianhlavac) [reply troubleshooting/index.adoc:6]: Já ne, :toc: pak budou na jednotlivých stránkách s problémy. Uvidíme kolik jich bude, zatím bych počítal s tím, že budou rozděleny do souborů.

File: troubleshooting/index.adoc (hunk 3 above)
Comment (marianhlavac) [reply troubleshooting/index.adoc]: Why?

File: troubleshooting/index.adoc (hunk 1 above)
Comment (hroncok) [reply troubleshooting/index.adoc]: let's not, it's a mix. we can add attribution page where we list all authors

File: troubleshooting/index.adoc (hunk 2 above)
Comment (hroncok) [reply troubleshooting/index.adoc:6]: rozumím, ok

File: troubleshooting/index.adoc (hunk 3 above)
Comment (hroncok) [reply troubleshooting/index.adoc]: looks better?

**Comment by hroncok** [comment]
Bump?

File: troubleshooting/index.adoc (hunk 1 above)
Comment (marianhlavac) [reply troubleshooting/index.adoc]: Nice thing, when defined author this way, is that on courses.fit.cvut.cz the author is displayed in the footer, even when someone else made the last commit. I think it should be used for someone who did most of the writing on that page. But the choice is yours ofc.

**Comment by marianhlavac** [comment]
https://github.com/3DprintFIT/BI-3DT/pull/108/commits/26b6cd0e75395eb7a5fbebabef15fede9983d045 expects the #109 will be merged soon.

**Review by hroncok — APPROVED** [review]

**Pull request merged**